app/components/Search/Search.css

```diff
@@ -41,18 +41,22 @@
 }
 
 .searchResultItemIcon {
-  margin-right: 15px;
-  margin-left: 3px;
+  color: var(--lego-font-color);
 }
 
 .resultsContainer {
   display: flex;
   flex-wrap: wrap;
-  max-height: calc(100vh - 120px);
+  max-height: calc(100svh - 120px);
```

Comment:
Cool! I didn't know this was a thing

Reply:
Yes I did not know about it either before today. [WebKit have added a bunch useful ones](https://webkit.org/blog/12445/new-webkit-features-in-safari-15-4/#solving-pain-points)